packages/core/documentation/src/components/DocumentationContent.vue

```diff
@@ -21,14 +20,9 @@
             </div>
           </template>
         </EntityEmptyState>
-        <DocumentationPageEmptyState
-          v-else
-          :can-edit="canEdit"
-          @create-documentation="handleAddClick"
-        />
       </KCard>
       <EntityEmptyState
-        v-else-if="enableV2EmptyStates"
+        v-else
```

Comment:
@afrzhussain : seems this `v-else` shouldn't be here as the original condition was `v-else-if="enableV2EmptyStates"` and we're removing the `enableV2EmptyStates`